### PR TITLE
Remember checkbox

### LINKED ISSUE #656: "Remember Me" style checkbox at login.
It would be nice to have a "Remember Me" style checkbox at login so users that do not want to login each time they visit the site could be remembered.

### PR DESCRIPTION
**Pull request by sijad**
resolves #656

let me know if the last commit is unnecessary or should be in another PR

### CODE REVIEW

File: js/forum/src/components/LogInModal.js [hunk 1]
```diff
@@ -59,6 +66,11 @@ export default class LogInModal extends Modal {
               disabled={this.loading} />
           </div>
 
+          <label className="checkbox">
+            <input name="remember" type="checkbox" bidi={this.remember} disabled={this.loading} />
+            {app.translator.trans('core.forum.log_in.remember_me_text')}
```
Comment (tobscure) [inline js/forum/src/components/LogInModal.js]: Please change the key to `remember_me_label`

**Comment by tobscure** [comment]
Awesome work @sijad! Just one little change before I'll merge.

**Comment by sijad** [comment]
all done

**Comment by tobscure** [comment]
Thanks! Can you PR flarum-ext-english too?

**Comment by sijad** [comment]
yes sure

**Comment by sijad** [comment]
sorry, I forgot to compile js after changing `remember_me_text` to `remember_me_label`

**Pull request merged**